<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>CESAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>INVERSO</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>250</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>INVERSO GRUPAL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cifrado:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCifradoCesar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDesplazamiento">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Texto Plano:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inverso:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCifradoInverso">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtPlano">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCifrado">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cifrar</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCifradoInversoGrupal">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inverso Grupal:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
